tests/tck/features/match/Base.feature

```diff
@@ -388,3 +388,34 @@ Feature: Basic match
       MATCH () --> (v) --> () return *
       """
     Then a ExecutionError should be raised at runtime: Can't solve the start vids from the sentence: MATCH ()-->(v)-->() RETURN *
+
+  Scenario: Return columns with same name
```

Comment:
We only check it when output to successor sentence.

Reply:
But `RETURN` is the last clause.

Reply:
I means you don't need check the output, the others sentence will check the input from match.